docs/pages/foundations/iconography/library.js

```diff
@@ -116,6 +116,15 @@ function findIconByCategory(icon?: string, category: string) {
   );
 }
 
+function iconHasKeyword(iconName?: string, searchTerm: string) {
```

Comment:
this part here could be optimize probably  mapping into key values rather but it works and for Docs doesnt seem a big deal!

Reply:
It's a bit more complicated logic so that it matches partial matches, so instead of keywords.includes, which would only match "create" but not "crea", it look through each keyword and then does include